engine/crates/runtime/src/operation_cache.rs

```diff
@@ -41,19 +34,19 @@ where
 // ---------------------------//
 // -- No-op implementation -- //
 // ---------------------------//
-impl HotCacheFactory for () {
+impl OperationCacheFactory for () {
     type Cache<V> = ()
 where
     V: Clone + Send + Sync + 'static + serde::Serialize + serde::de::DeserializeOwned;
 
-    async fn create<V>(&self, _: CachedDataKind) -> Self::Cache<V>
+    async fn create<V>(&self) -> Self::Cache<V>
```

Comment:
:bug: **Bug**
The `create` function implementation for `OperationCacheFactory` is missing the `kind: CachedDataKind` parameter. This could lead to incorrect cache creation if the `kind` was used to determine specific cache behaviors. If the `kind` parameter is still relevant, it should be reintroduced in the implementation.

```suggestion
    async fn create<V>(&self, kind: CachedDataKind) -> Self::Cache<V>
```